examples/wifi/scan/README.md

```diff
@@ -1,11 +1,11 @@
 # Wifi SCAN Example
 
-This example shows how to use scan of ESP32.
+This example shows how to use the scan functionality of ESP32.
```

Comment:
To complement my comment below I would update this sentence changing "... the scan functionality of ESP32." to "... the scan functionality of Wi-Fi driver of ESP32."

Reply:
Sounds even better. I will fix this asap (hopefully, since most likely I'll spend a few hours fiddling with the new ESP32 :P  )